src/KafkaNET.Library/KafkaNET.Library.nuspec

```diff
@@ -1,25 +1,14 @@
 <?xml version="1.0"?>
-<package >
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.Kafkanet</id>
-    <version>1.0.0.0</version>
-    <authors>Microsoft Corporation</authors>
-    <owners>Microsoft Corporation</owners>
-    <licenseUrl>https://github.com/Microsoft/Kafkanet/blob/master/LICENSE.txt</licenseUrl>
```

Comment:
The existing package metadata, including licenseUrl and requireLicenseAcceptance flags should not be deleted.